Conclusion Page Hifi

jaydhulia opened the pull request:
https://www.loom.com/share/4b179c82a2614871bac245e593476877

ahmedhamodi commented on app/public/static/css/completed_page.css:
```diff
@@ -0,0 +1,168 @@
+}
+
+.back-to-lobby-button {
+  /*width: 133px;*/
+  /*height: 44px;*/
+  /*padding-top: 3.90625vh;*/
```
Let's remove, unless it may be needed in the future.

ahmedhamodi reviewed (approved):
Code approval - looks good to me.

jaydhulia commented:
Cleaned up the code, added padding and added "Link Copied" button on click that changes back to "Copy Link" after 3 seconds. Updated loom video: https://www.loom.com/share/21fa470f082c4ae8a0552af7ece51307

ahmedhamodi reviewed (approved):
lgtm

jaydhulia commented:
> Could we perhaps remove the 3 second change back and keep it like "link copied" unless the other link copy is clicked?

Oops, accidentally closed. Updated loom video: https://www.loom.com/share/364a0218369d47f9a65319349a64e0e8

aaronabraham311 reviewed (approved):
Product approval
